recipes/genbooster/meta.yaml

```
@@ -0,0 +1,83 @@
+{% set name = "genbooster" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Techtonique/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: cd1582ea4064f39041fc47771effad2b0232118c065ecf8f99a87b41d6083023
+
+build:
+  number: 0
+  script: "{{ PYTHON }} setup.py build_ext --inplace && {{ PYTHON }} -m pip install . -vv"
+  skip: true  # [py < 39 or win]
+
+requirements:
+  build:
+    - rust >=1.70
+    - cargo-bundle-licenses
```

Please run this and add the bundled licenses in the license metadata.

```suggestion
    - cargo-bundle-licenses
```

---

Please don't mark as resolved until you do. See https://github.com/conda-forge/biome-feedstock/blob/69943908c497351596bb6cc34b53ce4d19cdbaf6/recipe/meta.yaml for an example of what you need to do in case you are lost.